src/app/ReadClient.cpp

```diff
@@ -717,5 +794,37 @@ CHIP_ERROR ReadClient::SendSubscribeRequest(ReadPrepareParams & aReadPreparePara
     return CHIP_NO_ERROR;
 }
 
-}; // namespace app
-}; // namespace chip
+void ReadClient::OnResubscribeTimerCallback(System::Layer * apSystemLayer, void * apAppState)
+{
+    ReadClient * const _this = reinterpret_cast<ReadClient *>(apAppState);
+    assert(_this != nullptr);
+    _this->SendSubscribeRequest(_this->mReadPrepareParams);
+    _this->mNumRetries++;
+}
+
+bool ReadClient::Resubscribe()
+{
+    bool shouldResubscribe = true;
+    mReadPrepareParams.mResubscribePolicy(mNumRetries, mIntervalMsec, shouldResubscribe);
```

Comment:
Why is `mIntervalMsec` a member?  It's only used in this function, no?

Reply:
just move it to local variable, thanks